packages/@webex/plugin-meetings/test/unit/spec/roap/request.ts

```diff
@@ -0,0 +1,113 @@
+import {assert} from '@webex/test-helper-chai';
+import sinon from 'sinon';
+import MockWebex from '@webex/test-helper-mock-webex';
+import Metrics from '@webex/plugin-meetings/src/metrics';
+
+import RoapRequest from '@webex/plugin-meetings/src/roap/request';
+
+
+describe('RoapRequest', () => {
+  describe('attachRechabilityData', () => {
+    let webex;
+
+    beforeEach(() => {
+      webex = new MockWebex();
+    });
+    
+    it('attaches the reachability data when it exists', async () => {
+      // @ts-ignore
+      const roapRequest = new RoapRequest({}, {parent: webex});
+
+      const sdp = {};
```

Comment:
it would be worth having this non-empty so that the test really verifies that attachRechabilityData() really attaches reachability to existing properties and not simply replaces the whole object with one that has just reachability results

Reply:
changed